src/verifications/verifications.spec.ts

```diff
@@ -393,6 +393,22 @@ describe('Verifications handler', function test() {
         .to.have.deep.property('rejectReason', { reason: 'denied' });
     });
 
+    it('correctly maps accounts to identities and vice versa', async () => {
+      const identity = await verifications.getIdentityForAccount(accounts[0], true);
+      const account = await verifications.getOwnerAddressForIdentity(identity);
+      expect(account).to.eq(accounts[0]);
+    });
+
+    it('finds the eventual owner account address for a given alias identity', async () => {
```

Comment:
Should we update the title due to logic changes?
```suggestion
    it('finds the owner identity of a given alias identity', async () => {
```